composer.json

```diff
@@ -14,6 +14,10 @@
 			"homepage": "https://typo3.org/community/"
 		}
 	],
+	"require": {
+		"typo3/cms-core": "^11.4 || ^11.5 || dev-master",
+		"typo3/cms-fluid-styled-content": "^11.4 || ^11.5 || dev-master"
```

Comment:
This should be reduced to `^11.4` which will automatically pull 11.5, 11.6 etc. OTOH `dev-master` should not be used by default since it will pull the very latest version (thus e.g. v12, v13, etc).